ManusGloves/include/ManusGloveHelper.h

```diff
@@ -0,0 +1,377 @@
+/**
+ * @file ManusGloveHelper.h
+ * @authors Ehsan Ranjbari <ehsan.ranjbari@iit.it>
+ * @copyright 2024 AMI Lab - Istituto Italiano di Tecnologia
+ *            Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ * @date 2024
+ */
+
+#ifndef MANUS_GLOVE_HELPER_HPP
+#define MANUS_GLOVE_HELPER_HPP
+
+#include <Eigen/Dense>
+
+// std
+#include <array>
+#include <iostream>
+#include <memory>
+#include <stdint.h>
+#include <mutex>
+#include <vector>
+#include <chrono> // Used for haptic commands.
+#include <cmath> // Used for rounding float values.
+#include <iomanip> // Used for printing glove data, and converting glove IDs to strings.
+
+// YARP
+#include <yarp/os/LogStream.h>
+#include <yarp/sig/Vector.h>
+
+// Manus Glove
+#include <ManusGlove.h>
+
+/**
+ * GloveControlHelper is an helper class for controlling the glove.
+ */
+namespace manusGlove
```

Comment:
This is because I wanted to follow Sense Glove (https://github.com/robotology/wearables/blob/2e9199dd8b450e23733aecc98d3488a504739d35/devices/HapticGlove/include/SenseGloveHelper.hpp#L27-L32) naming convention.

Reply:
I don't think we are forced to follow that pattern 😊

Reply:
> I don't think we are forced to follow that pattern 😊

of course not. I will apply your suggestion now then :)

Reply:
I also remembered another reason I used like that is this error:
```c++
error C2872: 'ManusGlove': ambiguous symbol [C:\yar
p-device-haptic-gloves\build\ManusGloves\ManusGlove.vcxproj]
```

The `ManusGlove` is the name of the class in cpp file.

Reply:
Ah I see, nevermind then!